### DISCUSSION TITLE
RESTEASY003880: Unable to find contextual data of type: org.keycloak.models.KeycloakSession

### DISCUSSION THREAD

**Discussion by sonic-sw** [keycloak/keycloak#19263] [Q&A]
Dear community,

i am struggling to start KC21.0.1 with mysql on K8s GKE. I am new to KC, please excuse me if I ask the obvious.
My deployment terminates after two of the below ERROR messages:

`ERROR [io.quarkus.resteasy] (executor-thread-1) org.jboss.resteasy.spi.LoggableFailure: RESTEASY003880: Unable to find contextual data of type: org.keycloak.models.KeycloakSession"`

I have build an image via Dockerfile:
```
FROM quay.io/keycloak/keycloak:21.0.1 as builder

# Enable health and metrics support
ENV KC_HEALTH_ENABLED=true
ENV KC_METRICS_ENABLED=false

# Configure a database vendor
ENV KC_DB=mysql
ENV KC_DB_URL=jdbc:mysql://IP:3306/keycloak
ENV KC_DB_USERNAME=****************
ENV KC_DB_PASSWORD=****************
ENV KC_TRANSACTION_XA_ENABLED=false

ENV KC_PROXY=edge
ENV KC_HOSTNAME=abc.def.com
ENV KC_HTTP_ENABLED=true

ENV KC_LOG_LEVEL=info

# Configure a admin
ENV KEYCLOAK_ADMIN=****************
ENV KEYCLOAK_ADMIN_PASSWORD=****************

WORKDIR /opt/keycloak

RUN /opt/keycloak/bin/kc.sh build
RUN /opt/keycloak/bin/kc.sh show-config

FROM quay.io/keycloak/keycloak:21.0.1
COPY --from=builder /opt/keycloak/ /opt/keycloak/

ENTRYPOINT ["/opt/keycloak/bin/kc.sh"]
```
and I start the server with `args: ["start --optimized"]`

Where do I go wrong?
Thank you in advance.

**Comment by yvmori1** [comment]
Did you ever find the solution?
I am facing the exact same issue.
